go.mod

```diff
@@ -54,11 +54,16 @@ require (
 	github.com/uber/jaeger-client-go v2.22.1+incompatible
 	github.com/urfave/negroni v1.0.0
 	go.opentelemetry.io/otel v0.2.1
-	golang.org/x/crypto v0.0.0-20200221231518-2aa609cf4a9d
+	golang.org/x/crypto v0.0.0-20200320181102-891825fb96df
 	golang.org/x/lint v0.0.0-20200302205851-738671d3881b // indirect
 	golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d
 	golang.org/x/sys v0.0.0-20200302150141-5c8b2ff67527 // indirect
 	golang.org/x/tools v0.0.0-20200313205530-4303120df7d8
-	gopkg.in/ini.v1 v1.54.0 // indirect
+	gopkg.in/gorp.v1 v1.7.2 // indirect
 	gopkg.in/square/go-jose.v2 v2.4.1
 )
+
+replace (
+	github.com/ory/fosite => github.com/aaslamin/fosite v0.21.6-0.20200324061735-29317aed4141
+	github.com/ory/x => github.com/aaslamin/x v0.0.8-0.20200324063550-0e5572e8199d
+)
```

Comment:
ℹ️FYI @aeneasr this is just temporary to get CI to build until:

- https://github.com/ory/x/pull/129
- https://github.com/ory/fosite/pull/402

have been merged 🚀